README.md

```diff
@@ -128,6 +132,8 @@ print(error)   # Output: len(5)+3 contains ill-typed subterm len(5)
 
 ```javascript
 import pyfinalo from 'pyfinalo_js';
+// or alias for conciseness
+import p from 'pyfinalo_js';
 
 // Direct evaluation
 const result = pyfinalo.show(pyfinalo.add(pyfinalo.int(5), pyfinalo.int(3)));
```

Comment:
update the term building callings to 'p' (that is, all except invocation to `pyfinalo.show`)